Website/src/items/renderings/edgewebsite/News/NewsGrid.yml

```diff
@@ -4,47 +4,47 @@ Parent: "deb2e94a-9a21-4305-ad48-76704da0e6f1"
 Template: "04646a89-996f-4ee7-878a-ffdbf1f0ef0d"
 Path: /sitecore/layout/Renderings/Project/edgewebsite/News/NewsGrid
 SharedFields:
-- ID: "003a72cd-4cd6-4392-9862-41d4159929cd"
-  Hint: Data source
-  Value: /sitecore/content/EdgeWebsite/home/news
-- ID: "037fe404-dd19-4bf7-8e30-4dadf68b27b0"
-  Hint: componentName
-  Value: NewsGrid
-- ID: "06d5295c-ed2f-4a54-9bf2-26228d113318"
-  Hint: __Icon
-  Value: "edgeicons/32x32/news-grid.png"
-- ID: "1a7c85e5-dc0b-490d-9187-bb1dbcb4c72f"
-  Hint: Datasource Template
-  Value: 
-- ID: "b0b15510-b138-470e-8f33-8da2e228aafe"
-  Hint: Rendering Contents Resolver
-  Value: "{FC5201C0-7E5F-483C-818B-BFE8A55A5A0F}"
-- ID: "b5b27af1-25ef-405c-87ce-369b3a004016"
-  Hint: Datasource Location
-  Value: 
+  - ID: "003a72cd-4cd6-4392-9862-41d4159929cd"
```

Comment:
Please revert formatting changes made to this file. Serialized YAML items have a specific format that must not be changed. Please disable "format on save" option of your text editor if that was the cause of the change.